### DISCUSSION TITLE
TeslaMate Sign-In bug

### DISCUSSION THREAD

**Discussion by ludomove74** [teslamate-org/teslamate#1401] [Q&A]
Dear All,

First, thank you for developping this App.
I successfully installed teslamate by following this tuorial : https://satheesh.net/2019/09/28/teslamate-digitalocean-docker-step-by-step-installation-guide/ in "Advanced mode" and set up 2 SSL certificate with 2 FQDN.
I set this up on an Ubuntu Server 2004 LTS.

Grafana is accessible with the HTTPS link and I can successfuly login, but i have a trouble to access to the teslamate admin interface. I succesfully login through the htaccess, then when I am on the login page, I cannot write anything, the login box seems the be frozen (tested on my PC and a mobile device).

Please find below a screenshot in the Dev Mode (Chrome - I blushed some variables on the image) :

![image](https://user-images.githubusercontent.com/79713970/109329113-804f4300-785a-11eb-9316-ddd954a23e1a.png)

Do you have any idea of what's going wrong ?

**Comment by ludomove74** [comment]
Finally works.

Do you know if Teslamate is multi users ?

**Reply by kimyd** [reply]
How did you solve the issue?
